src/docs/common/callouts/callout-mqtt-subscriber.adoc

```diff
@@ -0,0 +1 @@
+Micronaut MQTT implements the interface automatically because it is annotated with `@MqttSubscriber`. https://micronaut-projects.github.io/micronaut-mqtt/latest/guide/#consumer[It listens for messages].
```

Comment:
In the second sentence, what is the antecedent for "It"? Should this read ...

```suggestion
Because it is annotated with `@MqttSubscriber`, Micronaut MQTT implements the interface automatically and https://micronaut-projects.github.io/micronaut-mqtt/latest/guide/#consumer[listens for messages].
```